roles/network_plugin/calico/tasks/main.yml

```diff
@@ -208,14 +233,39 @@
   when: (not legacy_calicoctl and
          peer_with_router|default(false) and inventory_hostname in groups['kube-node'])
 
+- name: Calico | Configure peering with route reflectors
+  shell: >
```

Comment:
We do not use a separate file in all other `calicoctl` commands. I don't think we should start doing it here.